pom.xml

```diff
@@ -17,7 +17,7 @@
   <properties>
     <stack.version>3.9.3-SNAPSHOT</stack.version>
     <asciidoc.dir>${project.basedir}/src/main/asciidoc</asciidoc.dir>
-    <groovy.version>3.0.2</groovy.version>
+    <groovy.version>3.0.4</groovy.version>
```

Comment:
Why not 3.0.5 ?

Reply:
3.0.5 is throwing a ConcurrentModificationException at VertxTransformation. This wasn't happening until 3.0.4, so I guess it is a bug. I raised an issue about this at Groovy, expecting for a solution.

Reply:
Or maybe an workaround

Reply:
It builds now with 3.0.5